llumnix/backends/vllm/migration_backend.py

```diff
@@ -277,18 +274,16 @@ def do_recv(self, src_handle, blocks: List[int]):
                     col.recv(recv_cache, src_handle, self.group_name)
                 self.cache_engine.attn_backend.swap_blocks(recv_cache[cache_idx], self.gpu_cache[layer_idx], src_to_dst)
         self.migration_stream.synchronize()
-        self.put_back_cache(dummy_cache_idx)
 
 def get_migration_backend(migration_config: MigrationConfig, cache_engine: CacheEngine, worker_handle_list, scheduling_strategy,
                         is_driver_worker, gpu_cache, worker_rank, local_rank) -> MigrationBackendBase:
-    if cache_engine.num_gpu_blocks < migration_config.migration_buffer_blocks:
-        logger.warning("migration_buffer_blocks({}) is larger than num_gpu_blocks({}), reducing it to num_gpu_blocks."
-                       .format(migration_config.migration_buffer_blocks, cache_engine.num_gpu_blocks))
-        migration_config.migration_buffer_blocks = cache_engine.num_gpu_blocks
+    if cache_engine.num_gpu_blocks < migration_config.migration_cache_blocks:
+        logger.warning("migration_cache_blocks({}) is larger than num_gpu_blocks({}), reducing it to num_gpu_blocks."
+                       .format(migration_config.migration_cache_blocks, cache_engine.num_gpu_blocks))
+        migration_config.migration_cache_blocks = cache_engine.num_gpu_blocks
 
     target_migration_backend = None
     backend = migration_config.migration_backend
-    assert backend in ['nccl', 'gloo', 'rpc'], "Unsupported migration backend: {} for llumnix".format(backend)
```

Comment:
Keep this change, as the migration backend for vllm only supports these three types. Additionally, other migration backend keywords may also appear in the migration backend for other engines.